decidim-core/app/packs/stylesheets/decidim/modules/_typography.scss

```diff
@@ -1,338 +0,0 @@
-/* Variables */
-
-/* Typography - Foundation overrides */
-
-button,
-input {
-  font-family: inherit;
-}
-
-a {
-  color: $anchor-color;
-  font-weight: $anchor-font-weight;
-
-  &:hover {
-    color: $anchor-color-hover;
-    text-decoration: underline;
-    text-decoration-skip-ink: auto;
-  }
-
-  &:focus {
-    outline: $anchor-outline-focus;
-    outline-offset: $anchor-outline-offset;
-  }
-}
-
-button {
-  &:focus {
-    outline: $button-outline-focus;
-    // outline-color: var(--highlight);
-  }
-}
-
-@mixin heading($heading) {
-  line-height: 1.2;
-  font-size: (
-      map-get(map-get(map-get($header-styles, small), $heading), "font-size") *
-        0.0625
-    ) + em;
-
-  @include breakpoint(medium) {
-    font-size: (
-        map-get(map-get(map-get($header-styles, medium), $heading), "font-size") *
-          0.0625
-      ) + em;
-  }
-}
-
-.heading1 {
-  font-weight: 600;
-
-  @include heading(h1);
-
-  line-height: 1.1;
-}
-
-.subheading1 {
-  text-align: center;
-  font-size: 1.25rem;
-  margin: 0 auto;
-  max-width: 600px;
-  padding-bottom: 2rem;
-  margin-top: -3rem;
-}
-
-.heading2 {
-  @include heading(h2);
-}
-
-.heading3 {
-  @include heading(h3);
-}
-
-.heading4 {
-  @include heading(h4);
-}
-
-.heading5 {
-  @include heading(h5);
-}
-
-.heading6 {
-  text-transform: uppercase;
-  letter-spacing: 0.03em;
-  font-weight: 600;
-
-  @include heading(h6);
-}
-
-.heading-small {
-  font-size: 1rem;
-  line-height: $header-lineheight;
-}
-
-hr {
-  width: 50%;
-
-  &.reset {
-    width: 100%;
-  }
-}
-
-/* New typographic styles */
-
-// Introduced because .section-heading has a position: relative
-// that breaks when there are two headings in the same section
-// This should be revisited in the redesign.
-
-.heading-section-heading {
-  margin-bottom: 1rem;
-  font-weight: 600;
-  font-size: 1.125em;
-  letter-spacing: 0.05em;
-  text-transform: uppercase;
-
-  &::before {
-    content: "";
-    display: inline-block;
-    width: 4px;
-    height: 1.35em;
-    background-color: var(--primary);
-    margin-right: 0.5rem;
-    margin-bottom: -0.5rem;
-    vertical-align: top;
-  }
-
-  &.collapse {
-    margin: 0;
-  }
-
-  span {
-    font-weight: normal;
-  }
-}
-
-.section-heading {
-  position: relative;
-  margin-bottom: 1rem;
-  font-weight: 600;
-  font-size: 1.125em;
-  letter-spacing: 0.05em;
-  text-transform: uppercase;
-
-  &::before {
-    content: "";
-    display: inline-block;
-    width: 4px;
-    height: 1.35em;
-    background-color: var(--primary);
-    margin-right: 0.5rem;
-    margin-bottom: -0.5rem;
-    vertical-align: top;
-  }
-
-  &.collapse {
-    margin: 0;
-  }
-
-  span {
-    font-weight: normal;
-  }
-}
-
-.mini-title {
-  color: $muted;
-  font-size: 0.9rem;
-  letter-spacing: 0.01em;
-  text-transform: uppercase;
-  font-weight: 600;
-  margin-bottom: 0;
-
-  &__strong {
```

Comment:
Weirdly, this one I found it in a SCSS file (so it was migrated?), but I can't find the HTML 😕 

```shell
decidim-budgets/app/packs/stylesheets/budgets.scss
76:      .mini-title__strong {
```

Reply:
Removed